styles.css

```diff
@@ -511,13 +502,75 @@ hr {
   background-color: #0d151e;
 }
 
+.see-more {
+  padding: 1px 0;
+  width: 142px;
+}
+
+.see-more .see {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  text-decoration: none;
+  color: #396df2;
+  padding-right: 20px;
+}
+
+.mobile-project-container {
+  display: none;
+  z-index: 10;
+  position: fixed;
+  background: rgba(106, 110, 116, 0.7);
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  top: 0;
+}
+
+.popup-item-mob {
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+  background: #fff;
+  margin-top: 20px;
+  border-right-width: 20px;
+  margin-left: 20px;
+  margin-bottom: 20px;
+  border-radius: 10px;
+  padding: 0 15px;
```

Comment:
- [ ] [_OPTIONAL_] Great job using flex to layout this component, the position and width of this part is a little bit off from the design. Kindly fix it.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/169381285-6e4a2659-c347-4204-925c-9450ccb35850.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/169381444-50118102-51a4-4f3e-b84a-503ea9ed854a.png)

The Figma design is centred both horizontally and vertically.